### PR TITLE
Issue/#26 Add basic README

### LINKED ISSUE #26: Improve README
**Is your feature request related to a problem? Please describe.**

As a potential contributor or developer, I want to know how the repo is set up so I can get started on development quickly. Typically, this is done by looking at the project's README, but the one at the root of the project is blank.

**Describe the solution you'd like**
I want the project's README to describe the project structure and how I can get my development environment set up. While the README in the frontend folder is good, it only describes the frontend.

The new README should:
* Describe the CI setup (maybe including a build passing/failing badge)
* Describe how to get the project running
* Direct me to project documentation (e.g. the wiki and the README in the frontend directory)

**Describe alternatives you've considered**

Other alternatives may include having a 'getting started' page on the wiki, but it would probably be better to have some information in the README since this is the conventional solution. The wiki can go into further depth as required.

**Additional context**
Not applicable.

### PR DESCRIPTION
**Pull request by timovv**
Add a basic README that contains core information about the project,
such as where to go to get more information.

Resolves #26.

## Proposed Changes

  - Improve the current README so that it has some meaningful information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
-# se701-assignment1
+# Threader
```
Comment (timovv) [inline README.md]: Is this what we're calling it?

File: README.md [hunk 2]
```diff
@@ -1 +1,19 @@
...
+![Build: Frontend](https://github.com/se701-group6/se701-assignment1/workflows/CD_FRONTEND/badge.svg)
+
+Brought to you by SOFTENG 701 Group 6, Threader is a message board that lets users make text posts that other users
+can comment on.
+
+## Getting Started
+
+The application has two parts, the frontend and the server (or backend), which are in separate directories in this repository. Both parts need to be running for the application to work properly. To get everything set up, see:
+* [Backend tech page](https://github.com/se701-group6/se701-assignment1/wiki/Backend/)
+* [Frontend tech page](https://github.com/se701-group6/se701-assignment1/wiki/Frontend)
```
Comment (timovv) [inline README.md]: This page doesn't exist yet, but I linked it under the assumption it will be created soon.

Comment (ErnWong) [reply README.md]: I've created an empty page for it for now so the link works.

File: README.md (hunk 1 above)
Comment (ErnWong) [reply README.md]: We'll find out in 11pm when voting closes :ballot_box:

**Comment by JamesUOA** [comment]
Lgtm

File: README.md (hunk 1 above)
Comment (JamesUOA) [reply README.md]: I'll edit the logo based on the result

Comment (ErnWong) [reply README.md]: Threader it is

**Pull request merged**